Adjust GTM32 Env

Vertabreak opened the pull request:
This change corrects the crash when LCD knob is pressed on discount LCD.

Vertabreak committed:
Geeetech G2Pro configuration (#16553)

thisiskeithb committed:
Use Flash EEPROM on BTT002 (#16558)

thinkyhead committed:
Get test configs from GitHub

thinkyhead committed:
Fix typo

Vertabreak committed:
fix lcd issue

Vertabreak committed:
Merge pull request #80 from Vertabreak/Vertabreak-patch-1

fix lcd issue

Vertabreak committed:
Merge pull request #82 from MarlinFirmware/bugfix-2.0.x

Bugfix 2.0.x

Vertabreak committed:
Update platformio.ini

Vertabreak committed:
Update platformio.ini

Phr3d13 commented:
I can confirm that this change fixes build issues for the STM32F103VE_GTM32 environment in platformio.

Vertabreak committed:
Merge pull request #85 from MarlinFirmware/bugfix-2.0.x

Bugfix 2.0.x

Vertabreak committed:
Delete Configuration_adv.h

Vertabreak committed:
Delete Configuration.h

Vertabreak commented:
my bugfix branch seems to have become out of sync i rebased it so its clean now and i removed the files that didn't belong in this PR i just noticed it wasn't right.

Vertabreak commented:
im just gonna close this one and submit a clean copy.
